src/maintainer/pinning_deps.rst

```diff
@@ -67,6 +67,36 @@ If a package is not pinned in `conda-forge-pinning <https://github.com/conda-for
 There is additional documentation on this pinning scheme in `the conda docs <https://docs.conda.io/projects/conda-build/en/latest/source/variants.html#build-variants>`_.
 
 
+.. _what_the_pins_mean:
+
+What the pins mean
+==================
+
+Pins, generally, are all about setting benchmarked versions of packages or libraries that serve as dependencies for some other packages or libraries, just so these dependent packages behave as much as the author(s) of the dependent packages would want them to behave. 
+
+So conda-forge pins are about declaring the standard version (a specific version) of a package dependency that conda-forge or a `conda recipe <https://github.com/conda-forge/staged-recipes>`_ supports, and how recipes that depend on some specific version of some libraries are set in order to not use other varying versions of the dependency.
+
+.. _abis_and_the_conda-forge-pinning_feedstock:
+
+ABIs and the conda-forge-pinning feedstock
+==========================================
+
+conda-forge pinning can be expounded in terms of ABIs (Application Binary Interface), that is, as some programs require specific versions of some API libraries to function correctly - so do some packages require specific versions of some ABIs to function correctly, which is why some libraries used within the conda-forge feedstock are specific to some particular ABI versions.
```

Comment:
@bashirk Would you mind rewriting this paragraph, I needed to read twice to comprehend.

Reply:
Great! I'll work on rewriting this to be as comprehensive as possible. 

Many thanks for the feedback.